composer.json

```diff
@@ -99,14 +99,7 @@
         "symfony/form":                   "^2.7|^3.0|^4.0",
         "symfony/yaml":                   "^2.7|^3.0|^4.0",
         "symfony/templating":             "^2.7|^3.0|^4.0",
-
-        "psr/http-message":               "^1.0",
-        "php-http/client-implementation": "^1.0",
-        "php-http/httplug":               "^1.0",
-        "php-http/client-common":         "^1.3",
-        "php-http/message-factory":       "^1.0",
-        "php-http/discovery":             "^1.0",
-        "php-http/guzzle6-adapter":       "^1.1"
```

Comment:
If you mention a class of any of those packages in your code, the package should stay.

Reply:
Well, the corresponding bundle already manage that even if you are right.

This will be in any case mandatory if the bundle is made optional.

Reply:
@greg0ire fixed.

Reply:
Yeah I know but if it (the bundle) decides to drop a dependency, or to bump a requirement, and you are not ready for that requirement, you want to know.